### DISCUSSION TITLE
Which role is allowed to manage channels and root orgs (tenants) ?

### DISCUSSION THREAD

**Discussion by nidhikachhawa24** [Sunbird-Ed/Community#251] [Documentation] [answered]
Suppose, we want to manage channels and root orgs (tenants) details, then can any role make changes?
Or is there any specific role who can manage the channels and root orgs (tenants) changes?
Is there any way to delete a whole channel and root orgs (tenants)?

cc: @rajeevsathish @reshmi-nair @Krishnaj20

**Comment by Krishnaj20** [comment, marked as answer]
@reshmi-nair - can you respond to this query please?

@rhwarrier

**Reply by reshmi-nair** [reply]
@nidhikachhawa24 Org update api can be used to update tenant information, but the api is not exposed to public and changes can be made from backend service. There no way to delete the tenant orgs.

**Reply by chitranshu-keshav** [reply]
@nidhikachhawa24 Can you please mark this thread as answered if your query is resolved?
